streams/src/main/java/org/apache/kafka/streams/state/internals/InMemoryWindowStore.java

```diff
@@ -462,14 +460,21 @@ public boolean hasNext() {
             }
 
             final Bytes key = getKey(next.key);
-            if (key.compareTo(getKey(keyFrom)) >= 0 && key.compareTo(getKey(keyTo)) <= 0) {
+            if (isKeyWithinRange(key)) {
                 return true;
             } else {
                 next = null;
                 return hasNext();
             }
         }
 
+        private boolean isKeyWithinRange(final Bytes key) {
```

Comment:
Per comment above, can you make this method more readable by splitting the statements?